src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp

```diff
@@ -592,8 +611,29 @@ namespace AppInstaller::Manifest
                 { ERROR_SUCCESS_REBOOT_REQUIRED, ExpectedReturnCodeEnum::RebootRequiredToFinish },
                 { ERROR_SUCCESS_REBOOT_INITIATED, ExpectedReturnCodeEnum::RebootInitiated },
                 { ERROR_INSTALL_USEREXIT, ExpectedReturnCodeEnum::CancelledByUser },
+                { ERROR_BAD_CONFIGURATION, ExpectedReturnCodeEnum::ContactSupport },
+                { ERROR_INSTALL_UI_FAILURE, ExpectedReturnCodeEnum::ContactSupport },
+                { ERROR_CREATE_FAILED, ExpectedReturnCodeEnum::ContactSupport },
+                { ERROR_PRODUCT_VERSION, ExpectedReturnCodeEnum::AlreadyInstalled },
+                { ERROR_INSTALL_REJECTED, ExpectedReturnCodeEnum::BlockedByPolicy },
+            };
+        case InstallerTypeEnum::AdvancedInstaller:
+            // See https://www.advancedinstaller.com/user-guide/exe-setup-file.html
+            return
+            {
+                { ERROR_INSTALL_ALREADY_RUNNING, ExpectedReturnCodeEnum::InstallInProgress },
+                { ERROR_DISK_FULL, ExpectedReturnCodeEnum::DiskFull },
+                { ERROR_INSTALL_SERVICE_FAILURE, ExpectedReturnCodeEnum::ContactSupport },
+                { ERROR_SUCCESS_REBOOT_REQUIRED, ExpectedReturnCodeEnum::RebootRequiredToFinish },
+                { ERROR_SUCCESS_REBOOT_INITIATED, ExpectedReturnCodeEnum::RebootInitiated },
+                { ERROR_INSTALL_USEREXIT, ExpectedReturnCodeEnum::CancelledByUser },
+                { ERROR_BAD_CONFIGURATION, ExpectedReturnCodeEnum::ContactSupport },
+                { ERROR_INSTALL_UI_FAILURE, ExpectedReturnCodeEnum::ContactSupport },
+                { ERROR_CREATE_FAILED, ExpectedReturnCodeEnum::ContactSupport },
                 { ERROR_PRODUCT_VERSION, ExpectedReturnCodeEnum::AlreadyInstalled },
                 { ERROR_INSTALL_REJECTED, ExpectedReturnCodeEnum::BlockedByPolicy },
+                { 1, ExpectedReturnCodeEnum::CancelledByUser },

```

Comment:
```suggestion
                { -1, ExpectedReturnCodeEnum::CancelledByUser },
```
And 1 is "launched with wrong value".